tests/acceptance.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+setup() {
+# Create dockers and namespaces
+    DOCKER1="$(docker run --rm -d -it alpine sh)"
+    DOCKER2="$(docker run --rm -d -it alpine sh)"
+    sudo ip netns add NS1
+    sudo ip netns add NS2
+}
+
+teardown() {
+# Delete dockers and namespaces
+    docker rm -f $DOCKER1
+    docker rm -f $DOCKER2
+    sudo ip netns del NS1
+    sudo ip netns del NS2
+}
+
+
+
+@test "Docker .. Docker" {
+      sudo ./koko -d $DOCKER1,vethD1D2,10.10.10.1/29 \
```

Comment:
Could you please add IPv6 address as well, like that?
`-d $DOCKER1,vethD1D2,10.10.10.1/29,2001::1/64`

Reply:
I don't know if it's better to split the tests in IPv4 and IPv6 instead of testing both at same time because [docker has ipv6 disabled by default](https://docs.docker.com/engine/userguide/networking/default_network/ipv6/#how-ipv6-works-on-docker) and it will fail.

After enable the ipv6 flag in the docker daemon you can use ipv6 commands inside the containers but `koko`  fails with the following error:
```
 sudo ./koko -d $DOCKER1,vethD1D2,10.10.10.1/29,2001:DB8::1/64 -d $DOCKER2,vethD2D1,10.10.10.2/29,2001:DB8::2/64
Create veth...
veth add failed: failed to add IP addr 2001:db8::1/64 to "vethD1D2": permission denied
```
I recall several issues in the past with ipv6 and docker and we had to enable some kernel parameters, will try to dig more into this if I have more time. 

My docker version is 
`Docker version 17.09.0-ce, build afdb6d4`